scss/styles.scss

```diff
@@ -0,0 +1,78 @@
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: $font-big;
+  position: relative;
+  margin-bottom: 100px;
+  padding-left: 100px;
```

Comment:
the padding should be added to the body which is the element that wants to separate itself from its elements